### DISCUSSION TITLE
Uno.Extensions.Toolkit.UI and Uno.Extensions.Toolkit.WinUI have been deprecated, remove all references to this package?

### DISCUSSION THREAD

**Discussion by VincentH-Net** [unoplatform/uno#13674] [Q&A] [answered]
In a new solution created with the Uno VS Wizard, latest version, I get:
> Uno.Extensions.Toolkit.UI and Uno.Extensions.Toolkit.WinUI have been deprecated, remove all references to this package

However, the brand new sln wizard offered this as an option

Are these packages deprecated and if yes, what are recommended alternatives?

Thanks

**Comment by kazo0** [comment, marked as answer]
Hi @VincentH-Net!

Was this message shown immediately after creating a new Uno app with the wizard or did you then update any packages to their latest `-dev` versions?

I believe the `Uno.Extensions.Toolkit.UI` and `Uno.Extensions.Toolkit.WinUI` are indeed deprecated as of the 3.x-dev packages for Extensions since there is no longer a need for a dependency on Uno Toolkit for the theme switching service, so those packages were dropped. 

By default, the latest stable templates should be installing the stable `2.5.11` version of the Extensions packages where this should not yet be deprecated.

The latest (dev) templates should no longer include this dependency anymore.

Let me know if that helps or if perhaps there is a mix-up in the templates somewhere that needs to be corrected

**Comment by VincentH-Net** [comment]
Thanks for clarifying @kazo0  - it happened in the general chaos that ensued after upgrading to dev versions (as part of trying to resolve errors due to targeting .NET 8 RC1). I have decided to stay on .NET 7 until the next Uno / .NET release makes this work OOB.
